appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: "{build}"
+
+clone_folder: c:\gopath\src\github.com\manifoldco\torus-cli
+
+platform:
+  - x64
+
+environment:
+  GOPATH: c:\gopath
+  GOVERSION: 1.7
+  GLIDEVERSION: 0.12.3
+
+init:
+  - git config --global core.autocrlf input
+
+install:
+  - rmdir c:\go /s /q
+  - appveyor DownloadFile https://storage.googleapis.com/golang/go%GOVERSION%.windows-amd64.msi
+  - msiexec /i go%GOVERSION%.windows-amd64.msi /q
+  - set Path=c:\go\bin;c:\gopath\bin;%Path%
+  - go version
+  - go env
+  - appveyor DownloadFile https://github.com/Masterminds/glide/releases/download/v%GLIDEVERSION%/glide-v%GLIDEVERSION%-windows-amd64.zip
+  - md %GOPATH%\bin
+  - 7z e glide-v%GLIDEVERSION%-windows-amd64.zip -o%GOPATH%\bin *.exe -r
+
+build_script:
+  - c:\gopath\bin\glide.exe install
+
+test_script:
+  - FOR /f "" %%G in ('go list ./... ^| find /i /v "/vendor/"') DO (go test -run=. -bench=. -v %%G)
```

Comment:
By default `make` isn't available. There's a [Windows variant](http://gnuwin32.sourceforge.net/packages/make.htm) available but we'd have to install it. (I haven't tested this, so not sure how compatible it is with our Makefile)